docs/configuration.html

```diff
@@ -239,8 +239,8 @@
 
 <p>More details about consumer configuration can be found in the scala class <code>kafka.consumer.ConsumerConfig</code>.</p>
 
-<h4><a id="newconsumerconfigs" href="#newconsumerconfigs">3.3.2 New Consumer Configs</a></h4>
-Since 0.9.0.0 we have been working on a replacement for our existing simple and high-level consumers. The code is considered beta quality. Below is the configuration for the new consumer:
+<h4><a id="newconsumerconfigs" href="#newconsumerconfigs">3.3.2 Java Consumer Configs</a></h4>
+Since 0.9.0.0 we have been working on a replacement for our existing simple and high-level consumers. The code is considered beta quality. Below is the configuration for the Java consumer:
```

Comment:
This comment seems a little old. While it's true that we're still working on it, it's more or less a completed product. Maybe we just say that it was introduce in 0.9.0.0? Also, we should remove the mention of "beta quality." We merged a PR about this previously, but it looks like we missed this.